### PR TITLE
Update zh_cn.lang for next version

### PR DESCRIPTION
**Pull request by mc-kaishixiaxue**

### CODE REVIEW

**Comment by mezz** [comment]
Thank you!

@3TUSK can you review?

File: src/main/resources/assets/jei/lang/zh_cn.lang [hunk 1]
```diff
@@ -17,13 +17,17 @@ jei.tooltip.recipe.id=配方 ID: %s
...
+jei.tooltip.bookmarks.usage.key=将鼠标悬停在某个成分上，然后按“%s”将其添加为书签。
+jei.tooltip.bookmarks.not.enough.space=这里没有足够的空间来显示书签。
 
 # Error Tooltips
 jei.tooltip.error.recipe.transfer.missing=缺失物品
 jei.tooltip.error.recipe.transfer.inventory.full=物品栏已满
 jei.tooltip.error.recipe.transfer.unknown=未知错误，请查阅日志获得详细信息
 jei.tooltip.error.recipe.transfer.no.server=服务器必须安装 JEI
-jei.tooltip.error.recipe.transfer.too.large.player.inventory=配方不是 2×2 的
+jei.tooltip.error.recipe.transfer.too.large.player.inventory=配方不是 2x2 的
```
Comment (3TUSK) [inline src/main/resources/assets/jei/lang/zh_cn.lang]: Revert this. That `×` is `U+00D7, &#215; MULTIPLICATION SIGN`.

File: src/main/resources/assets/jei/lang/zh_cn.lang [hunk 2]
```diff
@@ -58,6 +64,8 @@ config.jei.interface=用户界面
 config.jei.interface.comment=用户界面的相关设定。
 config.jei.interface.overlayEnabled=启用物品列表
 config.jei.interface.overlayEnabled.comment=在打开的 GUI 一侧显示物品列表。
+config.jei.interface.bookmarkOverlayEnabled=显示书签
+config.jei.interface.bookmarkOverlayEnabled.comment=在打开的 GUI 旁边显示书签列表。
```
Comment (3TUSK) [inline src/main/resources/assets/jei/lang/zh_cn.lang]: Either change this one (as following), or change the above one (`在打开的 GUI 一侧显示物品列表` -> `在打开的 GUI 旁边显示物品列表`).
```suggestion
config.jei.interface.bookmarkOverlayEnabled.comment=在打开的 GUI 一侧显示书签列表。
```

File: src/main/resources/assets/jei/lang/zh_cn.lang [hunk 3]
```diff
@@ -17,13 +17,17 @@ jei.tooltip.recipe.id=配方 ID: %s
 jei.tooltip.not.enough.space=屏幕空间不足，无法显示 JEI 界面。
 jei.tooltip.ingredient.list.disabled=JEI 悬浮界面已禁用。
 jei.tooltip.ingredient.list.disabled.how.to.fix=按 %s 以启用。
+jei.tooltip.bookmarks=JEI 书签
+jei.tooltip.bookmarks.usage.nokey=在“按键”设置中为 JEI 书签添加按键设置。
+jei.tooltip.bookmarks.usage.key=将鼠标悬停在某个成分上，然后按“%s”将其添加为书签。
```
Comment (3TUSK) [inline src/main/resources/assets/jei/lang/zh_cn.lang]: No quotation marks.
```suggestion
jei.tooltip.bookmarks.usage.key=将鼠标悬停在某个成分上，然后按 %s 将其添加为书签。
```

File: src/main/resources/assets/jei/lang/zh_cn.lang [hunk 4]
```diff
@@ -17,13 +17,17 @@ jei.tooltip.recipe.id=配方 ID: %s
 jei.tooltip.not.enough.space=屏幕空间不足，无法显示 JEI 界面。
 jei.tooltip.ingredient.list.disabled=JEI 悬浮界面已禁用。
 jei.tooltip.ingredient.list.disabled.how.to.fix=按 %s 以启用。
+jei.tooltip.bookmarks=JEI 书签
+jei.tooltip.bookmarks.usage.nokey=在“按键”设置中为 JEI 书签添加按键设置。
```
Comment (3TUSK) [inline src/main/resources/assets/jei/lang/zh_cn.lang]: image
```suggestion
jei.tooltip.bookmarks.usage.nokey=在按键设置中为 JEI 书签添加快捷键。
```

or

```suggestion
jei.tooltip.bookmarks.usage.nokey=在“控制…”中为 JEI 书签添加按键设置。
```

I prefer the first one.

File: src/main/resources/assets/jei/lang/zh_cn.lang [hunk 5]
```diff
@@ -40,6 +44,8 @@ key.jei.recipeBack=显示上个查看的配方
 key.jei.toggleCheatMode=作弊模式开关
 key.jei.previousPage=上一页
 key.jei.nextPage=下一页
+key.jei.bookmark=添加/删除一个成分的书签
+key.jei.toggleBookmarkOverlay=显示/隐藏加入书签的成分
```
Comment (3TUSK) [inline src/main/resources/assets/jei/lang/zh_cn.lang]: 除了“成分”还能怎么翻译？“成分”听上去很奇怪。

**Review by 3TUSK — CHANGES_REQUESTED** [review]
I was summoned.

**Comment by mc-kaishixiaxue** [comment]
The translation improvement was completed.

**Review by 3TUSK — APPROVED** [review]

**Comment by mezz** [comment]
Thank you both!

**Pull request merged**